frontends/ui/src/app/globals.css

```
@@ -312,3 +312,837 @@ body {
+  border-radius: 3px;
+  background: var(--border-color-base);
+}
+.thinking-reasoning-text {
+  margin: 0;
+  padding: 0.45rem 0.7rem;
+  color: var(--text-color-subtle);
+  font-size: 0.74rem;
+  font-style: italic;
+  line-height: 1.55;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)